drf_link_navigation_pagination/__init__.py

```diff
@@ -13,6 +16,12 @@ def _eval_str_as_boolean(value: str):
     return str(value).lower() in ("true", "1", "t", "y")
 
 
+class BadLimitValue(APIException):
+    status_code = status.HTTP_400_BAD_REQUEST
+    default_detail = _("Bad limit value sent.")
+    default_code = "bad_limit_value"
```

Comment:
**suggestion:** Following our [playbook](https://github.com/juntossomosmais/playbook/blob/master/backend/python.md#exce%C3%A7%C3%B5es), perhaps we could move it to an **exceptions** module.